### PR TITLE
tick_until_end resets vel after it ends

### PR DESCRIPTION
**Pull request by ichumuh**
fixes a bug where giskard would continue with the velocity from the previous MSC, if the last one didn't set it to 0

### CODE REVIEW

File: test/test_motion_statechart/test_json_parsing.py [hunk 1]
```diff
@@ -347,3 +348,39 @@ def test_cancel_motion():
...
+    msc_copy = MotionStatechart.from_json(new_json_data, **kwargs)
+
+    kin_sim = Executor(
+        world=pr2_world,
+        controller_config=QPControllerConfig.create_default_with_50hz(),
+    )
+
+    kin_sim.compile(motion_statechart=msc_copy)
+
+    kin_sim.tick_until_end()
```
Comment (maltehue) [inline test/test_motion_statechart/test_json_parsing.py:386]: shouldnt you assert that world statre acc, vel and jerk are zero, or do you not want to test that here?

Comment (ichumuh) [reply test/test_motion_statechart/test_json_parsing.py:386]: this test was actually supposed to catch a different bug, that i forgot to mention in the description. it was a bug fix of exception parsing

Comment (maltehue) [reply test/test_motion_statechart/test_json_parsing.py:386]: then maybe add a test that checks if zero velocity, acc and jerk holds at the end of a motion?

Comment (ichumuh) [reply test/test_motion_statechart/test_json_parsing.py:386]: yeah, i'm testing this implicitly in the other test. but can't hurt to check it explicitly.

Comment (ichumuh) [reply test/test_motion_statechart/test_json_parsing.py:386]: done
